### PR TITLE
[REF] inventory: move removal strats into own folder

### PR DESCRIPTION
**Pull request by Felicious**
Move the removal strategy docs out of advanced operations and into its own directory.

### CODE REVIEW

**Comment by StraubCreative** [comment]
@samueljlieber can you run sanity checks here and ensure everything is proper before JCS's review?

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+:nosearch:
+
+==================
+Removal strategies
+==================
+
+.. toctree::
+   :titlesonly:
+
+   removal_strategies/removal
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst:10]: I think you can actually transplant the contents of `removal.rst` into `removal_strategies` since it's purpose was to be a general/config doc for the other ones. If you think so too then can move the contents, delete the old file, and add a redirect to point here 👍

**Review by StraubCreative — COMMENTED** [review]
@Felicious comment below for your consideration.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst (hunk 1 above)
Comment (jcs-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst:10]: good catch :)

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @Felicious, this is just a quick review on the restructure before passing to JCS! Coincidentally, I was just reading thought this documentation yesterday and thought that this restructure was needed, so great timing! 

[I agree](https://github.com/odoo/documentation/pull/7530#discussion_r1465759025) with ZST and JCS that `removal_strategies.rst` can contain the contents of `removal.rst`:

```
removal_strategies.rst (removal.rst contents)
├── closest_location.rst
├── fefo.rst
├── fifo.rst
├── least_packages.rst
└── lifo.rst
```
You are also [missing redirects](https://github.com/odoo/documentation/blob/17.0/redirects/MANUAL.md) for each of the moved files, please add them to the [redirects/17.0.txt](https://github.com/odoo/documentation/blob/17.0/redirects/17.0.txt) file.

Once these edits are made, tag me once more so I can review before I pass it along 🙂
Thank you!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst [hunk 2]
```diff
@@ -1,7 +1,20 @@
+:nosearch:
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst:1]: No need to include the `:nosearch:` meta tag because `removal_strategies.rst` now has content that we want searchable (instead of only a TOC). 
```suggestion
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst [hunk 3]
```diff
@@ -15,11 +28,11 @@ Odoo automatically select how products are selected for orders:
    :stub-columns: 1
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst:28]: This is out of scope of this PR, but want to note that this table does not format well on mobile. Maybe in another [IMP] PR this could be adjusted? I could see a card layout being useful here, the [Accounting and Invoicing](https://www.odoo.com/documentation/master/applications/finance/accounting.html) doc is a good example.

File: redirects/17.0.txt [hunk 4]
```diff
@@ -2,3 +2,28 @@
...
+applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fefo.rst
+applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fefo.rst
+
+# closest locations
+applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/closest_location.rst
+applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/closest_location.rst
+
+# least packages
+applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/least_packages.rst
+applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/least_packages.rst
```
Comment (samueljlieber) [inline redirects/17.0.txt:29]: Each redirect should be on one line, grouped by the app/module, and a comment summarizing the redirect can be added at the end of the redirect line 🙂

```suggestion
# inventory_and_mrp/inventory

applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/closest_location.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/closest_location.rst   # advanced_operations_warehouse/closest_location.rst --> removal_strategies/closest_location.rst
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fefo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fefo.rst                           # advanced_operations_warehouse/fefo.rst --> removal_strategies/fefo.rst
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fifo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fifo.rst                           # advanced_operations_warehouse/fifo.rst --> removal_strategies/fifo.rst
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/least_packages.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/least_packages.rst       # advanced_operations_warehouse/least_packages.rst --> removal_strategies/least_packages.rst
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/lifo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/lifo.rst                           # advanced_operations_warehouse/lifo.rst --> removal_strategies/lifo.rst
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst                             # advanced_operations_warehouse/removal_strategies.rst --> removal_strategies/removal_strategies.rst

```

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, thank you for implementing the changes! I have just a couple of structural fixes, please see below for reference. I will push up a commit with these changes so we can get this PR in structure review 🙂 I am approving my review of the structure, I will still need to do a full technical review.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst (hunk 1 above)
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst]: Applied the suggestion!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst [hunk 5]
```diff
@@ -1,7 +1,19 @@
...
 
+.. toctree::
+   :titlesonly:
+
+   removal_strategies/fifo
+   removal_strategies/lifo
+   removal_strategies/fefo
+   removal_strategies/closest_location
+   removal_strategies/least_packages
+
```
Comment (jcs-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst:16]: minor suggestion: put the toctree at the end of the document, as it is usually what we do on pages that have both content to display and children pages.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/least_packages.rst [hunk 6]
```diff
@@ -6,7 +6,7 @@ The *Least Packages* removal strategy fulfills an order by opening the fewest nu
 which is ideal for maintaining organized stock without needing to open multiple boxes.
 
 .. seealso::
-   - :doc:`Other removal strategies <removal>`
+   - :doc:`About removal strategies <../removal_strategies>`
```
Comment (jcs-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/least_packages.rst:9]: Technically, changing the wording would be part of another commit since it could also be applied to older versions than 17.0. But this is such a minor change that you can disregard my comment. It's mostly to keep in mind that when we apply different types of changes, they may apply to different versions.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fefo.rst [hunk 7]
```diff
@@ -6,7 +6,7 @@ The *First Expired, First Out* (FEFO) removal strategy targets products for remo
 assigned removal dates.
 
 .. seealso::
-   :doc:`Other removal strategies <removal>`
+   :doc:`About removal strategies <../removal_strategies>`
```
Comment (jcs-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fefo.rst:9]: same here as my other comment

File: redirects/17.0.txt [hunk 8]
```diff
@@ -2,3 +2,12 @@
 
 developer/reference/backend/views.rst developer/reference/user_interface/view_records.rst
 developer/reference/user_interface/view_architecture.rst developer/reference/user_interface/view_architectures.rst
+
+# inventory_and_mrp/inventory
```
Comment (jcs-odoo) [inline redirects/17.0.txt:6]:
```suggestion
# applications/inventory_and_mrp
```

It's only per section, and not per app.

File: redirects/17.0.txt [hunk 9]
```diff
@@ -2,3 +2,12 @@
...
+applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fifo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fifo.rst                           # advanced_operations_warehouse/fifo.rst --> removal_strategies/fifo.rst
+applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/least_packages.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/least_packages.rst       # advanced_operations_warehouse/least_packages.rst --> removal_strategies/least_packages.rst
+applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/lifo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/lifo.rst                           # advanced_operations_warehouse/lifo.rst --> removal_strategies/lifo.rst
+applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst                             # advanced_operations_warehouse/removal_strategies.rst --> removal_strategies/removal_strategies.rst
```
Comment (jcs-odoo) [inline redirects/17.0.txt:13]: - You can use `*` to indicate that the filename doesn't change, but rather it's location. Otherwise, it makes the comment as long to read as the rule itself ^^
- The last comment wasn't correct.

```suggestion
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/closest_location.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/closest_location.rst   # advanced_operations_warehouse/* --> removal_strategies/*
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fefo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fefo.rst                           # advanced_operations_warehouse/* --> removal_strategies/*
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fifo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/fifo.rst                           # advanced_operations_warehouse/* --> removal_strategies/*
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/least_packages.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/least_packages.rst       # advanced_operations_warehouse/* --> removal_strategies/*
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/lifo.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies/lifo.rst                           # advanced_operations_warehouse/* --> removal_strategies/*
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst                             # advanced_operations_warehouse/removal --> warehouses_storage/removal_strategies
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hi!

Nice work! :) I left a few really minor comments, but you can directly merge this once addressed.

@robodoo delegate+

Cheers!

**Comment by Felicious** [comment]
@robodoo r+

**Comment by StraubCreative** [comment]
@robodoo retry

**Comment by Felicious** [comment]
Rebase the PR again, as a sanity check and see if this will merge

**Comment by Felicious** [comment]
@robodoo retry

**Comment by Felicious** [comment]
@robodoo r+

**Comment by jcs-odoo** [comment]
Hi! It seems you forgot to include some changes in the commit you pushed, at least the ones you marked as resolved for the redirects.txt file.

Cheers

**Comment by Felicious** [comment]
Hi @jcs-odoo ! Sorry for missing the changes. I took a look at the [17.0.text](https://github.com/odoo/documentation/blob/a2fc9c6be558336f8d79c4eba13140a3c08012e1/redirects/17.0.txt) file and I couldn't find how it differs from your [comment](https://github.com/odoo/documentation/pull/7530#discussion_r1474760086). I'm so sorry, but could you elaborate on what I missed?

**Comment by jcs-odoo** [comment]
Apologies, you are right @Felicious ! I opened the wrong tab and missed one of the forced-pushed changes. 

Thanks again for your work
